examples/Demo/Shared/Pages/DataGrid/Examples/DataGridTypical.razor

```diff
@@ -1,21 +1,41 @@
 ﻿@inject DataSource Data
 
+<p>To test set ResizeType on the DataGrid to either DataGridResizeType.Discrete or DataGridResizeType.Exact</p>
+<p>Remove the parameter completely to get the original behavior</p>
 
-<FluentDataGrid Items="@FilteredItems" ResizableColumns=true Pagination="@pagination" GridTemplateColumns="0.2fr 1fr 0.2fr 0.2fr 0.2fr 0.2fr" RowClass="@rowClass" RowStyle="@rowStyle" Style="height: 405px;overflow:auto;">
+<FluentDataGrid Items="@FilteredItems" ResizableColumns=true ResizeType="DataGridResizeType.Exact" Pagination="@pagination" GridTemplateColumns="0.2fr 1fr 0.2fr 0.2fr 0.2fr 0.2fr" RowClass="@rowClass" RowStyle="@rowStyle" Style="height: 405px;overflow:auto;">
```

Comment:
The use of `ResizeType="DataGridResizeType.Discrete"` fails when the increment buttons are clicked.

![peek_3](https://github.com/microsoft/fluentui-blazor/assets/8350694/e89a4415-d97a-46a1-84d8-2a570b4c84f8)